src/main/java/ro/vladutit/Don/t/forget/v2/controller/ItemController.java

```diff
@@ -46,7 +46,7 @@ public String viewDashboardAll(Model item, Model category) {
     }
 
     //update an item by id
-    @RequestMapping("/updateItemForm/{id}")
+    @RequestMapping("/itemEdit/{id}")
```

Comment:
Here initially I wrote _/ item / edit / ..._ but I noticed that when I ran the application, the html page from this route, no longer displayed correctly (the images did not appear to me, the spacing was no longer correct) and after much research , I discovered that this extra "/" was the problem and then I came to the _/ itemEdit / ..._ variant. If you could explain to me why this is happening, it would be great.

Reply:
to have to name the route in the right way, I would check the docs. it should be /item/edit

Reply:
Okay. I will also research why the frontend part bothers me when I put the route in this way. Thank you!

Reply:
Ok. So in the last two days I've done intense research. I found it quite difficult to find information on this subject. It doesn't write clearly anywhere what is the most correct thing to do, I didn't see any clear steps or suggestions, but I saw suggestions where to use these routes and then I followed several examples, I even looked for bigger projects, I looked in source code to see how they did there and I changed the approach a bit. And with this approach, I no longer have problems with display on the frontend. I updated to the current commit. Please check now if it's ok.  Thank you.

**Source links:** 

1. https://docs.spring.io/spring/docs/4.0.x/spring-framework-reference/html/mvc.html
2. https://spring.io/blog/2009/03/08/rest-in-spring-3-mvc/
3. https://github.com/spring-projects/spring-framework/issues/10427
4. http://milanov.github.io/2015/10/13/routing-options-in-spring-mvc.html
5. https://github.com/resthub/springmvc-router
6. https://www.playframework.com/documentation/1.2.4/routes
7. https://stackoverflow.com/questions/14363559/application-routes-with-spring-mvc